corehq/warehouse/loaders/domain.py

```diff
@@ -0,0 +1,64 @@
+from corehq.apps.domain.models import Domain
+from corehq.warehouse.const import DOMAIN_STAGING_SLUG, DOMAIN_DIM_SLUG
+from corehq.warehouse.dbaccessors import get_domain_ids_by_last_modified
+from corehq.warehouse.etl import HQToWarehouseETLMixin, CustomSQLETLMixin
+from corehq.warehouse.loaders.base import BaseStagingLoader, BaseLoader
+from corehq.warehouse.models import DomainStagingTable, DomainDim
+from dimagi.utils.couch.database import iter_docs
+
+
+class DomainStagingLoader(HQToWarehouseETLMixin, BaseStagingLoader):
+    """
+    Represents the staging table to dump data before loading into the DomainDim
+
+    Grain: domain_id
+    """
+    slug = DOMAIN_STAGING_SLUG
+    model_cls = DomainStagingTable
+
+    @classmethod
+    def dependencies(cls):
+        return []
+
+    @classmethod
+    def field_mapping(cls):
+        return [
+            ('_id', 'domain_id'),
+            ('name', 'domain'),
+            ('default_timezone', 'default_timezone'),
+            ('hr_name', 'hr_name'),
+            ('creating_user_id', 'creating_user_id'),
+            ('project_type', 'project_type'),
+
```

Comment:
nit: a comment for the groups being done here will be helpful.

Reply:
not sure what the groups is - will remove